manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml

```diff
@@ -36,10 +36,9 @@ spec:
           else
             URL_ONLY_KUBECONFIG=/etc/kubernetes/kubeconfig
           fi
-          exec /cluster-controller-manager-operator
-        args:
-        - --leader-elect
-        - "--images-json=/etc/cloud-controller-manager-config/images.json"
+          exec /cluster-controller-manager-operator \
```

Comment:
I think we don't need `exec` here. It's possible just to call the binary as `/cluster-controller-manager-operator \`

Reply:
I copied this over from CNO, what difference does it make to the operation? Have you tested this without the exec? Otherwise I'll spin up a cluster and have a play

Reply:
yep, I tested it without `exec`. Basically it doesn't matter whether we have it or not, so it's fine to keep it as-is.

Reply:
Ok, let's keep as is for now as I know that's been tested, we can change later if we find a need to

Reply:
i think `exec` is proper here, i have to imagine CNO uses it because it will take over the shell that is running the process instead of spawning a new one. which makes exit and cleanup fairly easy. as long as we don't have anything else to run (and i don't think we do), `exec` is probably preferred here. just my 2 cents